.gitignore

```diff
@@ -56,3 +56,4 @@ venv/
 *.pyc
 __pycache__/
 
+Cargo.lock
```

Comment:
> Remove/Ignore Cargo.lock is unacceptable. We need to limit the dependencies to ensure a controlled build, please restore directly instead of regenerating.

The Cargo.lock is generated on local environment, why spread this  file to all envs?Is it enough to focus on the Cargo.toml?

May be this is answer.

FYR:
https://github.com/rust-lang/cargo/issues/315

cool